src/parcsr_ls/par_amg.c

```diff
@@ -12,6 +12,7 @@
  *****************************************************************************/
 
 #include "_hypre_parcsr_ls.h"
+#include "protos.h"
```

Comment:
@oseikuffuor1 See earlier comment on this.

Reply:
@oseikuffuor1 Do we need to include "protos.h"? Is  "_hypre_parcsr_ls.h" enough?

Reply:
@liruipeng No we don't. Thanks for catching that. I will correct that.